Blog: Is Google Analytics HIPAA compliant? (#2875)
PostHog/posthog.com: contents/blog/is-google-analytics-hipaa-compliant.md

- **Business Associate:** A third party that receives and / or manages data on behalf of a Covered Entity 

- **Business Associate Agreement:** An agreement between a covered entity and third party that handles their PHI; it ensures the Business Associate shares the same legal requirements and liability as the Covered Entity   

## Why isn't Google Analytics HIPAA compliant?

Because Google doesn't allow Covered Entities to enter into a BAA with it, as this [disclaimer](https://support.google.com/analytics/answer/6366371#zippy=%2Cin-this-article) on its website explains:

> Unless otherwise specified in writing by Google, Google does not intend uses of Google Analytics to create obligations under the Health Insurance Portability and Accountability Act, as amended, (“HIPAA”), and makes no representations that Google Analytics satisfies HIPAA requirements. If you are (or become) a Covered Entity or Business Associate under HIPAA, you may not use Google Analytics for any purpose or in any manner involving Protected Health Information unless you have received prior written consent to such use from Google.

> Because using Google Analytics requires transmitting data to Google-owned servers, a BAA would be necessary to be compliant. Were you to use Google Analytics to process or transmit PHI you would be liable for investigation and a fine.

Reviewer: Maybe swap the clauses round so we don't have two paras in a row starting with 'because'

> ## How much does violating HIPAA cost?

Reviewer: 'cost' makes it sound like you pay to violate HIPAA - maybe 'can I get fined for HIPAA breaches'? 

And in the explainer itself, it is worth maybe clarifying that smaller business can get hit as well - this isn't just for huge businesses to think about

> ## How much does violating HIPAA cost?

Author: ```suggestion
## HIPAA fines, explained
```

> HIPAA fines operate on a sliding scale based on the severity of the breach and the total number of breaches. This means they get expensive very quickly, especially for products or businesses with large user bases. The largest HIPAA fine to date is [$16 million](https://www.hipaajournal.com/anthem-inc-settles-state-attorneys-general-data-breach-investigations-and-pays-48-2-million-in-penalties/) against health insurer Anthem.

Author: ```suggestion
HIPAA fines operate on a sliding scale based on the severity of the breach and the total number of breaches. This means they get expensive very quickly, especially for products or businesses with large user bases. The largest HIPAA fine to date is [$16 million](https://www.hipaajournal.com/anthem-inc-settles-state-attorneys-general-data-breach-investigations-and-pays-48-2-million-in-penalties/) against health insurer Anthem.

Fines aren't limites to large businesses, either. In 2017, a [children's charity was fined](https://www.hhs.gov/hipaa/for-professionals/compliance-enforcement/agreements/ccdh/index.html) when it was revealed it stored data on a third-party platform without a BAA.
```

## HIPAA-compliant alternatives to Google Analytics

While Google Analytics is sufficient if you just want to see what pages a user views on a website, it won't help you improve your product in a meaningful way. For that, you need product analytics.

Product analytics platforms like [PostHog](https://posthog.com/) include features like [session recording](https://posthog.com/product/session-recording), [heatmaps](https://posthog.com/product/heatmaps) and [funnels](https://posthog.com/product/funnels) that help you understand how users navigate your product — not just what they're looking at.

If you want to find [product market fit](https://posthog.com/blog/how-to-product-market-fit), your product's [North Star metric](https://posthog.com/blog/north-star-metrics) or what your [users really need](https://posthog.com/blog/how-to-work-out-what-users-need)... you need product analytics.

While there are lots options out there, we think **PostHog is the most HIPAA-compliant product analytics platform** out there.

> Unlike third-party analytics options like Mixpanel or Amplitude, **PostHog is the only comprehensive product analytics suite you can self-host** on your own infrastructure. 

Reviewer: Does this claim sufficiently address the Matomo/Piwik objection? Maybe 'platform' instead of 'suite'

Author: Can change, but what was the objection?

Reviewer: You could argue that PostHog isn't the only comprehensive product analytics suite you can self-host, as you can self-host Matomo. 

I suppose you could say 'ah yes, but Matomo isn't comprehensive or a suite', but I think saying 'platform' makes us sound broader and more differentiated.

This is important because:

1. **Data never leaves your infrastructure:** Any time you export data out of your ecosystem, you're taking a risk. Covered Entities can be legally liable for mistakes made by their third-party Business Associates, and even when they're not the reputational risk is shared.

2. **You don't need a Business Associate Agreement:** BAAs are time-consuming and expensive, and also require frequent legal review to ensure continued compliance. Self-hosting means you don't need to enter into any additional BAAs, other than the one you should already have with your existing cloud provider.

> 3. **Compliance doesn't cost extra:** If you need HIPAA compliance, third-party providers will frequently demand you use a more expensive tier of their product than you really need. Customers who use PostHog as their [HIPAA-compliant analytics provider](link to listicle here) pay the same as customers with less stringent privacy needs.

Author: ```suggestion
3. **Compliance doesn't cost extra:** If you need HIPAA compliance, third-party providers will frequently demand you use a more expensive tier of their product than you really need. Customers who use PostHog as their [HIPAA-compliant analytics provider](https://posthog.com/blog/best-hipaa-compliant-analytics-tools) pay the same as customers with less stringent privacy needs.
```

> Read our [PostHog & HIPAA compliance guide](link to new docs page) for more info on how to deploy PostHog correctly, or [book a demo](https://posthog.com/book-a-demo) to learn more.

Author: ```suggestion
Read our [PostHog & HIPAA compliance guide](/docs/privacy/hipaa-compliance) for more info on how to deploy PostHog correctly, or [book a demo](https://posthog.com/book-a-demo) to learn more.
```

---

> date: 2022-21-02

Author: ```suggestion
date: 2022-02-21
```

title: Is Google Analytics HIPAA compliant?
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
featuredImage: ../images/blog/hipaa-compliant-product-analytics.png
featuredImageType: full
author: ["andy-vandervell"]
categories: ["Privacy", "Guides"]

> ---

Reviewer 2: (All of this is personal opinion, don't take it to heart, it has _nothing_ to do with quality of the content):

I'm guessing this is a heavily SEO optimised post, and that's the goal?

I hope we don't publish something like this on HN, because if I saw this on HN, I'd flag the shiz out of it.

----

So, before I start dishing out opinions, could use some guidance on what kind of review are you looking for? :)

Author: Yeah, this will go nowhere near HN or any social for that matter.

A BAA is necessary because Google Analytics transmits data to Google-owned servers when you use it. Were you to use Google Analytics to process or transmit PHI you would be liable for investigation and a fine.

## HIPAA fines, explained

HIPAA fines operate on a sliding scale based on the severity of the breach and the total number of breaches. This means they get expensive very quickly, especially for products or businesses with large user bases. The largest HIPAA fine to date is [$16 million](https://www.hipaajournal.com/anthem-inc-settles-state-attorneys-general-data-breach-investigations-and-pays-48-2-million-in-penalties/) against health insurer Anthem.

Fines aren't limited to large businesses. In 2017, a [children's charity was fined](https://www.hhs.gov/hipaa/for-professionals/compliance-enforcement/agreements/ccdh/index.html) due to storing PHI on a third-party platform without a BAA.

## HIPAA-compliant alternatives to Google Analytics

While Google Analytics is sufficient if you just want to see what pages a user views on a website, it won't help you improve your product in a meaningful way. For that, you need product analytics.

> Product analytics platforms like [PostHog](https://posthog.com/) include features like [session recording](https://posthog.com/product/session-recording), [heatmaps](https://posthog.com/product/heatmaps) and [funnels](https://posthog.com/product/funnels) that help you understand how users navigate your product — not just what they're looking at.

Author: ```suggestion
Product analytics platforms like [PostHog](https://posthog.com/) include features like [Session Recording](https://posthog.com/product/session-recording), [Heatmaps](https://posthog.com/product/heatmaps) and [Funnels](https://posthog.com/product/funnels) that help you understand how users navigate your product — not just what they're looking at.
```

While there are lots options out there, we think **PostHog is the most HIPAA-compliant product analytics platform** out there.

Unlike third-party analytics options like Mixpanel or Amplitude, **PostHog is the only comprehensive product analytics platform you can self-host** on your own infrastructure. 

This is important because:

1. **Data never leaves your infrastructure:** Any time you export data out of your ecosystem, you're taking a risk. Covered Entities can be legally liable for mistakes made by their third-party Business Associates, and even when they're not the reputational risk is shared.

2. **You don't need a Business Associate Agreement:** BAAs are time-consuming and expensive, and also require frequent legal review to ensure continued compliance. Self-hosting means you don't need to enter into any additional BAAs, other than the one you should already have with your existing cloud provider.

> 3. **Compliance doesn't cost extra:** If you need HIPAA compliance, third-party providers will frequently demand you use a more expensive tier of their product than you really need. Customers who use PostHog as their [HIPAA-compliant analytics provider](https://posthog.com/blog/best-hipaa-compliant-analytics-tools) pay the same as customers with less stringent privacy needs.

Author: ```suggestion
3. **Compliance doesn't cost extra:** If you need HIPAA compliance, third-party providers will frequently demand you use a more expensive tier of their product than you really need. Customers who use PostHog as their [HIPAA-compliant analytics](https://posthog.com/blog/best-hipaa-compliant-analytics-tools) provider pay the same as customers with less stringent privacy needs.
```
